packages/app-project/src/screens/ProjectAboutPage/ProjectAboutPageConnector.js

```diff
@@ -0,0 +1,62 @@
+import { observer, MobXProviderContext } from 'mobx-react'
+import React, { useContext } from 'react'
+import { arrayOf, bool, object, shape, string } from 'prop-types'
+
+import ProjectAboutPage from './ProjectAboutPage'
+
+/**
+  This is for enzyme tests.
+*/
+function useStoreContext (testStore) {
+  const { store } = testStore || useContext(MobXProviderContext)
+  return {
+    store
+  }
+}
+
+/**
+  Connect the about page to the store. Pass down correct aboutPages data.
+*/
+function ProjectAboutPageConnector ({ pageType, teamArray, testStore }) {
+  const { store } = useStoreContext(testStore)
+  const { inBeta } = store.project
+  const aboutPages = store.project.about_pages
```

Comment:
```suggestion
  const { 
    store: {
      project: {
        inBeta,
        about_pages
      }
    }
  } = useStoreContext(testStore)
```

You can do nested destructuring in cases like this. But you have to be careful of this if there aren't default values for each property, which can be a gotcha. You can define default values in the destructuring to deal with this (see: https://medium.com/@pyrolistical/destructuring-nested-objects-9dabdd01a3b8). Our store model should define default values as needed, however. Up to you how you want to write this.